hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/state/EidasCountrySelectedState.java

```diff
@@ -18,6 +18,7 @@
     private String countryEntityId;
     private final Optional<String> relayState;
     private List<LevelOfAssurance> levelsOfAssurance;
+    private final Boolean forceAuthenticationForNonEidasJourney;
```

Comment:
So how come we don't implement this on a base class? Couldn't we implement `forceAuthentication` on `AbstractState`? To ask this question another way, how many states now _don't_ have a `forceAuthentication` property?